lib/ansible/modules/net_tools/nmcli.py

```diff
@@ -501,10 +501,9 @@
 
 try:
     import gi
-    gi.require_version('NMClient', '1.0')
-    gi.require_version('NetworkManager', '1.0')
+    gi.require_version('NM', '1.0')
 
-    from gi.repository import NetworkManager, NMClient
+    from gi.repository import NM
```

Comment:
Is this change backward compatible ?

Reply:
The use of "gi" appears to create a module dependency on the remote host, which is really something that I think Ansible should avoid. So far I've been unable to get either the original nmcli module or this patch working on a RHEL6 remote host. Perhaps there's a better way?

Reply:
I would try/except to allow for older systems to continue working with the old lib, thus ensuring backwards compatibility